java-ast/addon/src/main/java/org/jboss/windup/ast/java/ReferenceResolvingVisitor.java

```diff
@@ -111,7 +113,7 @@ private void resolveCurrentTypeNames(CompilationUnit compilationUnit)
             {
                 typeLine = typeLine.substring(typeDeclaration.getJavadoc().toString().length());
             }
-            classReferences.add(new ClassReference(fqcn, ResolutionStatus.RESOLVED, TypeReferenceLocation.TYPE,
```

Comment:
This seems to be reverting some changes. Is this the result of a bad rebase or something?